src/cli/bin.js

```diff
@@ -16,6 +16,12 @@ updateNotifier({
 const cli = yargs
   .commandDir('commands')
   .demandCommand(1)
+  .fail((msg, err, yargs) => {
+    if (err) throw err // preserve stack
+    console.error(yargs.help())
```

Comment:
Here maybe you could use the [showHelp()](https://github.com/yargs/yargs#showhelpconsolelevelerror) function instead.